src/encoded/static/components/objectutils.js

```diff
@@ -5,6 +5,17 @@ import { Modal, ModalHeader, ModalBody, ModalFooter } from '../libs/bootstrap/mo
 import * as globals from './globals';
 
 
+// Display information on page as JSON formatted data
+export function displayAsJson(){
```

Comment:
Missing space, this showed up in codacy.  I don't always look at it but it's good to check.

Reply:
Also resolved this in the most recent commit.